torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -26,6 +27,8 @@ def _quantize_tensor(tensor, qtype):
         )
     if (qtype == DQuantType.FP16):
         return _fp32_to_fp16_with_clamp(tensor)
+    elif (qtype == DQuantType.BFP16):
+        return torch.ops.q.FloatToBfloat16Quantized(tensor)
```

Comment:
nit: do we want it to be a public API? If not, should we prefix it with `_` for the time being?